flink-tests/src/test/java/org/apache/flink/test/streaming/experimental/CollectITCase.java

```diff
@@ -38,33 +35,23 @@
  * <p>This experimental class is relocated from flink-streaming-contrib. Please see package-info.java
  * for more information.
  */
-public class CollectITCase extends TestLogger {
+public class CollectITCase extends AbstractTestBase {
 
```

Comment:
It seems that we do not use anything in AbstractTestBase class. Why do we still extend this class ?

Reply:
One thing I found the difference from them , If we extends AbstractTestBase class. No logs will print to console.

Reply:
Please have a look at the javadocs for AbstractTestBase.

The logs should still be printed as `AbstractTestBase` extends `TestBaseUtils` which extends `TestLogger`.

Reply:
Thanks @zentol.